server/src/main.rs

```diff
@@ -1,19 +1,46 @@
 mod routes;
 
-use std::io;
+use std::{env, io};
 
 use actix_files::Files;
-use actix_web::{middleware::Logger, App, HttpServer};
+use actix_session::{storage::CookieSessionStore, SessionMiddleware};
+use actix_web::{cookie::Key, middleware::Logger, web, App, HttpServer};
+use dotenv::dotenv;
 
+/// Returns a secret key that is used to encrypt the session cookie stored by the client.
+///
+/// # Panics
+///
+/// Panics if the environment variable `SESSION_SECRET` is not a base64-encoded string of at least
+/// 32 randomly-generated bytes.
+fn get_session_secret() -> Key {
+    let session_secret =
+        env::var("SESSION_SECRET").expect("environment variable `SESSION_SECRET` must be defined");
```

Comment:
The `dotenv` crate provides a `var` function that we can use here, so we don't need to `use std::env`

Reply:
This also guarantees that the `.env` is loaded at the time we look up the value.

Reply:
Changed in https://github.com/connorjayr/sust/pull/3/commits/807aec22384c91cb99e3b126cec730ae560bc5b4.